### ISSUE TITLE
List of people/orgs to include

### ISSUE THREAD

**Issue by jbenet** [ipfs/POST#4]
Other people + orgs will care about POST, either because of similar or related work in the {past, present, future}.
- W3C
- OpenAnnotation
- XMPP group
- Matrix.org
- Ethereum / Consensys group

Let's get 30+ more.

**Comment by harlantwood** [comment]
@jasnell, editor of the Activity Streams 2.0 spec:
- http://www.w3.org/TR/activitystreams-core/ 
- https://github.com/jasnell/w3c-socialwg-activitystreams

**Comment by ghost** [comment]
Probably also Tent: https://tent.io/docs

**Comment by RichardLitt** [comment]
- Bocoup. Specifically, @boazsender.
- Crosscloud.org

**Comment by boazsender** [comment]
Yeah, I'd love to know how @bocoup can support your work on distributing the web.

**Comment by jbenet** [comment]
hey @boazsender o/ :)

**Comment by boazsender** [comment]
Hi @jbenet \o/
